about.md

```diff
@@ -20,6 +20,10 @@ Servo is written in [**Rust**](https://www.rust-lang.org/), taking advantage of
 
 Since its creation in 2012, Servo has contributed to W3C and WHATWG web standards, reporting specification issues and submitting new cross-browser automated tests, and core team members have co-edited new standards that have been adopted by other browsers. As a result, the Servo project helps drive the entire web platform forward, while building on a platform of reusable, modular technologies that implement web standards.
 
+## WPT progress
+
+<iframe src="https://wpt.servo.org/?embed" scrolling="no" style="width: 100%; height: 411px;"></iframe>
+
```

Comment:
I think we need to add some kind of short paragraph about what is this. We could even have a `servo.org/wpt` page under *About* (like *Governance*) and have there an explanation, the chart embedded, and links to wpt.servo.org and wpt.fyi with Servo information. WDYT?

Reply:
done in af6ce1d89224e2fa57ec537b21ad1cb32a2a779b, though i think having it on the same page is ok for now :)